src/main/deploy/states/stateData.dtd

```diff
@@ -21,7 +21,7 @@
                             SHOOTER_OFF | SHOOT_LOWGOAL | MANUAL_SHOOT | ADJUSTHOOD | PREPARETOSHOOT | SHOOT_HIGHGOAL_CLOSE | SHOOT_HIGHGOAL_FAR | 
                             CLIMBER_OFF | CLIMBER_MANUAL | CLIMBER_INITIALREACH | CLIMBER_RETRACT | CLIMBER_RELEASE | CLIMBER_REACHTOBAR | CLIMBER_ROTATEOUT | CLIMBER_ROTATEIN | CLIMBER_HOLD |
                             LIFT_LIFT | LIFT_LOWER | LIFT_OFF |
-                            INDEXER_INDEX | INDEXER_EXPEL | INDEXER_OFF |
+                            INDEXER_INDEX | INDEXER_EXPEL | INDEXER_OFF | SERVO_CLOSED_CLOSED | SERVO_CLOSED_OPEN | SERVO_OPEN_CLOSED | SERVO_OPEN_OPEN |
```

Comment:
Please add mechanism to the state names so there is context as there are other mechanisms with servos